apps/nextjs/src/components/ContextProviders/ChatProvider.tsx

```diff
@@ -329,17 +300,17 @@ export function ChatProvider({
   useEffect(() => {
     if (toxicModeration) {
       setMessages([]);
-      setLessonPlan({});
+      //setLessonPlan({}); #TODO what to do here
```

Comment:
We need to wipe local state (but not wipe it in the db)

Reply:
Resolved this with an override